about.html

```diff
@@ -0,0 +1,98 @@
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+            </button>
+          </div>
+          <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+            <ul class="nav navbar-nav">
+              <li><a href="index.html">Home</a></li>
+              <li><a href="Gallery.html">Gallery</a></li>
+              <li class="active"><a href="about.html">About <span class="sr-only">(current)</span></a></li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+
+      <!--Main section -->
+      <div class="AboutMainSection">
+        <div class="container-fluid">
+          <div class="row">
+            <div class="col-md-12">
+              <div class="information">
+                <img class="banner" src="https://files1.porsche.com/filestore/image/multimedia/none/aboutporsche-xle/normal/1e37c200-6414-11e3-8f5f-001a64c55f5c/porsche-normal.jpg" alt="">
+              </div>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-md-12">
+              <h2>The Porshe Principle</h2>
+              <p class="AboutInfo">This quote gets to the heart of everything that makes Porsche what it is. As a brand, as a company and as an automotive manufacturer. It has been our guiding star – for more than 65 years. And it covers all the values that characterise our work and our vehicles. It’s no wonder, therefore, that no-one can describe this better than the person who created the very first sports car to bear the Porsche name: Ferdinand Anton Ernst - or Ferry Porsche, for short. In the beginning I looked around and could not find quite the car I dreamed of. So I decided to build it myself. Ferry Porsche His dream of the perfect sports car has always driven us – throughout our history. And we get closer to achieving it every day. With every concept, every development and every model. Along the way, we follow a plan, an ideal that unites us all. We refer to it simply as the Porsche Principle. The underlying principle is to always get the most out of everything. From day one, we have strived to translate performance into speed – and success – in the most intelligent way possible. It's no longer all about horsepower, but more ideas per horsepower. This principle originates on the race track and is embodied in every single one of our cars. We call it "Intelligent Performance".</p>
+            </div>
+          </div>
+        </div>
+          <div class="row">
+              <div class="col-md-12">
+                <video
+                 src="https://www.porsche.com/filestore/video/multimedia/none/principle-porsche/video-mp4/ebc94c1c-b4df-11e3-ab3e-001a64c55f5c/The-Porsche-Principle.mp4" autoplay poster="posterimage.jpg" controls loop>
+                </video>
+                  </div>
+              </div>
+          </div>
+          <!--The footer   -->
+          <footer id="myFooter">
```

Comment:
Don't leave in something like this that looks like a navbar if it doesn't actually function. Make it functional or take it out.